CONTRIBUTING.md

```diff
@@ -4,6 +4,59 @@ Government employees, public and members of the private sector are encouraged to
+
+```
+## April 20, 2024
+- Fixed bug that didn't allow users to post [DESENG-123](https://apps.itsm.gov.bc.ca/jira/browse/DESENG-123)
+```
+
+Take note of the previous date in the changelog when committing. If the current date is at the top of the file, add a bullet for your changes under that date header.
+
+## Pull Requests
+
+If a high volume of changes is to be submitted as a PR, we ask that the work be segmented into multiple PRs. A common threshold is that a PR should contain 20 changed files at most. Of course it also depends on the complexity of the changes but we ask that work be broken up into smaller logical chunks whenver possible. 
+
+### Examples
+
+1. Let's say you've added 5 new files and made complex changes to 10 others. You also have 5 files that have minor changes in syntax as a result of running the linter. This would be a good place to submit a PR.
+2. You have 1 new file added and you may need to make minor changes to syntax or variable names in 25-30 other files. This would be fine to submit as a single PR.
+
+Segmenting a PR is more of an art than a science and PRs typically won't be rejected if they're larger than 20 files. How you segment may be informed by how you choose to branch in git. An example segmentation workflow is provided below.
+
+### Segmentation technique
+
+One common technique used for segmenting PRs is to create a "parent" feature branch off of main. From there, you can create many "child" branches from the parent. A PR is submitted each time you want to merge a child into its parent. A child branch will represent one logical division of the work and, as mentioned above, will contain no more than 20 changed files (depending on complexity of changes).
+
+![PR Segmentation Diagram](docs/pr-segmentation.png)
+
+With this method of segmentation, you don't need to worry about committing incomplete or broken features to main. When all child branches have been completed for the feature, the parent is then merged back into main in a final PR. This final review can contain your CHANGELOG.md entry. In the title or description of your PR, please indicate that all work has already been reviewed.
+
+## Merge Techniques
+
+Due to how PRs are commonly segmented in this repository, and due to its active nature, rebasing is discouraged as a way to merge changes. Traditional merge commits are preferred as they preserve history and make it easier to resolve merge conflicts.
+
+## Guidelines for reviewers
+
+Conditions for whether to Approve or Request Changes for a PR can come with a lot of nuance. We try to air on the side of Approve unless there are obvious changes needed. Below is a list of reasons to "stop" a PR. Items with a star next to them are subject to interpretation by the review. Some items may prompt to discussion between the reviewer and the submitting developer so that choices can be justified.
+
+Examples of when to Request Changes
+- Logging functions are left in that aren't required or needed later
```

Comment:
Perhaps carve out an explicit exception for `console.error` if we're going forward with that